### PR TITLE
 BaSP-TG-64: get and delete super admins

### PR DESCRIPTION
**Pull request by julietapreliasco**
Get and delete data functionalities were updated using Redux. Spinner shared component implemented.

### CODE REVIEW

File: src/Components/SuperAdmins/index.js [hunk 1]
```diff
@@ -43,6 +34,7 @@ const SuperAdmins = () => {
     <>
       <div className={styles.container}>
         <h1>Super Admins</h1>
+        <Spinner isLoading={isLoading} />
```
Comment (lmalvasia) [inline src/Components/SuperAdmins/index.js:37]: We should show the only the spinner when isLoading is true.
```
if (isLoading) {
  return <Spinner isLoading={isLoading} />;
}
```

Comment (julietapreliasco) [reply src/Components/SuperAdmins/index.js:37]: Thanks!

**Review by lmalvasia — APPROVED** [review]

File: src/redux/superAdmins/actions.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import {
+  GET_SUPER_ADMINS_PENDING,
+  GET_SUPER_ADMINS_SUCCESS,
+  GET_SUPER_ADMINS_ERROR,
+  DELETE_SUPER_ADMINS_PENDING,
```
Comment (baudraccoverona) [inline src/redux/superAdmins/actions.js:5]:
```suggestion
  DELETE_SUPER_ADMIN_PENDING,
```

File: src/redux/superAdmins/actions.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  DELETE_SUPER_ADMINS_SUCCESS,
```
Comment (baudraccoverona) [inline src/redux/superAdmins/actions.js:6]:
```suggestion
  DELETE_SUPER_ADMIN_SUCCESS,
```

File: src/redux/superAdmins/actions.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  DELETE_SUPER_ADMINS_ERROR
```
Comment (baudraccoverona) [inline src/redux/superAdmins/actions.js:7]:
```suggestion
  DELETE_SUPER_ADMIN_ERROR
```

File: src/redux/superAdmins/actions.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+};
+
+export const getSuperAdminsError = (error) => {
+  return {
+    type: GET_SUPER_ADMINS_ERROR,
+    payload: error
+  };
+};
+
+export const deleteSuperAdminsPending = () => {
```
Comment (baudraccoverona) [inline src/redux/superAdmins/actions.js:30]:
```suggestion
export const deleteSuperAdminPending = () => {
```

File: src/redux/superAdmins/actions.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  return {
+    type: DELETE_SUPER_ADMINS_PENDING
+  };
+};
+
+export const deleteSuperAdminsSuccess = (payload) => {
```
Comment (baudraccoverona) [inline src/redux/superAdmins/actions.js:36]:
```suggestion
export const deleteSuperAdminSuccess = (payload) => {
```

File: src/redux/superAdmins/constants.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+export const GET_SUPER_ADMINS_PENDING = 'GET_SUPER_ADMINS_PENDING';
+export const GET_SUPER_ADMINS_SUCCESS = 'GET_SUPER_ADMINS_SUCCESS';
+export const GET_SUPER_ADMINS_ERROR = 'GET_SUPER_ADMINS_ERROR';
+export const DELETE_SUPER_ADMINS_PENDING = 'DELETE_SUPER_ADMINS_PENDING';
```
Comment (baudraccoverona) [inline src/redux/superAdmins/constants.js:4]: Same as above for this delete constants

**Review by baudraccoverona — APPROVED** [review]

**Review by lmalvasia — APPROVED** [review]

**Pull request merged**